simple_functions.py

```diff
@@ -2,6 +2,15 @@
 
 def double_number(a):
     return a+a
-
 def square_number(a):
     return a*a
+
+
+a = 5
+print(f'value before double_number(): {a}')
+result = double_number(a)
+print(f'value after double_number(): {result}')
+
+print(f'value before square_number(): {a}')
+result = square_number(a)
+print(f'value after square_number(): {result}')
```

Comment:
Have these things inside the function. make those changes and request for a re review

Reply:
@kavinmatheswara-ta  I corrected the changes waiting for your review